server/lib/src/driver.dart

```diff
@@ -43,6 +43,10 @@ class FlutterDriver {
     return _session;
   }
 
+  Session? getSession() {
```

Comment:
I believe the previous method should return `Session` instead of `Session?`

Reply:
Yup, we can update the return type of `getSessionOtThrow` to `Session`. i will fix it in a separate patch.